Deceive/MainController.cs

```diff
@@ -37,20 +39,22 @@ internal MainController(bool isLeague)
             _trayIcon.ShowBalloonTip(5000);
 
             // Create overlay and start following the LCU with it.
-            if (isLeague)
-            {
-                var process = Process.GetProcessesByName("LeagueClientUx").First();
-            
-                _overlay = new LCUOverlay();
-                _overlay.Show();
-                _follower = new WindowFollower(_overlay, process);
-                _follower.StartFollowing();
-            }
-            
+            if (isLeague) CreateOverlay();
+
             LoadStatus();
             UpdateUI();
         }
 
+        private void CreateOverlay()
+        {
+            var process = Process.GetProcessesByName("LeagueClientUx").First();
```

Comment:
We still occasionally have the problem where this will throw if the UX is very slow to start. Should we bother handling this, now that we're changing it anyway?

Reply:
What would you suggest? 
A: Not creating the overlay at all when the UX isn't there at this point, or 
B: retry to create the overlay like every 5 seconds or so?

Reply:
I'll accept a PR doing A, but if you have the time and motivation feel free to do B ;)

Reply:
Added B.